org.eclipse.cpacep/src/org/eclipse/cpacep/util/CPACEPConnector.java

```diff
@@ -71,7 +75,7 @@ private void initializeBaseCommandLine() {
 		baseCli.add(" -stats");
 	}
 
-	private String executeCommand(String command) {
+	public String executeCommand(String command) {

```

Comment:
Is there a reason for this to be public, despite testing?
If you put the test in the same package as this class, you can keep the method package-private.

Reply:
(Actually, I think there's no reason for CPACEP to be in the `util` package. But let's do that outside of this pull request to keep it clean).

Reply:
Hi Thomas,
We need to export CPACEPConnector class as a package in eclipse pde. Check `MANIFEST.MF` in cpapcep. I think we can keep the method as public without any reason.

Reply:
Sorry, I don't see the connection.
The method was `private` before and CPACEP worked, so why shouldn't it be possible to make it package-private now?
You should never make a method public without any reason.